targets/esp8266/docs/ESP-PREREQUISITES.md

```diff
@@ -30,7 +30,7 @@ sudo chown $USER /opt/Espressif/
 dependencies for x64
 ```
 sudo apt-get install git autoconf build-essential gperf \
-    bison flex texinfo libtool libncurses5-dev wget \
+    bison flex texinfo libtool libtool-bin libncurses5-dev wget \
```

Comment:
x86 and x64 deps are too repetitive. there should be no need for maintaining two almost identical copies of lists and steps. just present it once. and add the arch-dependent steps afterwards. (if I see it right, it's only the libc6-dev-i386 vs libc6-dev-amd64 package)

while ate it, fix the indentation of the line (this line is off by one space, next line is OK)

Reply:
I see your point, I'm going to reorder these deps.